pep-0683.rst

```diff
@@ -204,7 +204,7 @@ Performance
 
 A naive implementation shows `a 4% slowdown`_.  We have demonstrated
```

Comment:
I noticed on reddit, btw, that some people who read my Language Summit article were also slightly confused about whether immortal objects would make things faster or slower :) https://www.reddit.com/r/Python/comments/unx80h/comment/i8c2q7p/

Reply:
> Not sure if the specific number matters too much

I included a concrete number because it is the primary concern about the change.  The specific base number isn't critical except that it is larger than the threshold folks are comfortable with.

Regardless, I'll clarify the point.